seed_data/usersData.js

```diff
@@ -26,7 +26,8 @@ var usersData = {
       "_id"       : "58cc4941fc13ae612a10000a",
       "username"  : "Robb Elkson",
       "password"  : "perdonamierastil",
-      "email"     : "summonmonster@erastil.er"
+      "email"     : "summonmonster@erastil.er",
+      "freelancer": ["58cc4942fc13ae612c00004c","58cc4942fc13ae612c000043"]
```

Comment:
do these freelancer have proper verified status and owner set? or are they inconsistent on purpose for testing?